ajax/libs/jquery-parallax/package.json

```diff
@@ -12,5 +12,9 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/IanLunn/jQuery-Parallax"
-  }
+  },
+  "licenses": [
+    "MIT",
+    "GPL-1.0"
```

Comment:
`GPL-1.0` should be `GPL-3.0` according to https://github.com/IanLunn/jQuery-Parallax#jquery-parallax
> Dual licensed under the MIT and GPL licenses: http://www.opensource.org/licenses/mit-license.php http://www.gnu.org/licenses/gpl.html